tools/ansible/tasks/logging/test/logging_test.go

```diff
@@ -1,18 +1,26 @@
 package test
 
 import (
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
 	"os/exec"
 	"strings"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 )
 
+// Pining down version of Ruby Devkit to 2.5, 64 bit version
+// Ruby installation path kept default C:\
```

Comment:
Nit: not sure what you mean by this.

Reply:
There is an option to specify installation directory in the playbook, so I have mentioned the installation path I am assuming here